stl/inc/system_error

```diff
@@ -95,19 +103,43 @@ public:
 
 #ifdef __cpp_lib_concepts
     _NODISCARD strong_ordering operator<=>(const error_category& _Right) const noexcept {
-        return compare_three_way{}(_Addr, _Right._Addr);
+#ifndef _M_CEE_PURE
+        return _Bit_cast<uintptr_t>(_Addr) <=> _Bit_cast<uintptr_t>(_Right._Addr);
+#else // ^^^ !defined(_M_CEE_PURE) / defined(_M_CEE_PURE) vvv
+        return _Addr <=> _Right._Addr;
+#endif // ^^^ defined(_M_CEE_PURE) ^^^

```

Comment:
Fortunately, `/clr:pure` is forever incompatible with C++20, so we don't need to handle it for spaceship.